index.js

```diff
@@ -24,6 +24,11 @@ async function main() {
     process.argv.push(`--config-path=${configPath}`);
   }
 
+  if (!yargs.argv.throttlingMethod) {
+    // Disable simulation.
+    process.argv.push('--throttling-method=provided');
```

Comment:
Should line 29 be left in (just removing the "if", so that it runs 100% of the time)?

Reply:
The purpose of the if statement is to allow this setting to be overridden if the flag already exists. Without it, providing `--throttling-method=simulate` would actually send both arguments as `simulate,provided`.

Reply:
Ok, just checking :)